lms/djangoapps/course_goals/models.py

```diff
@@ -33,12 +33,16 @@ class Meta:
 
     user = models.ForeignKey(User, blank=False, on_delete=models.CASCADE)
     course_key = CourseKeyField(max_length=255, db_index=True)
+    # The goal a user has set for the number of days they want to learn per week
+    days_per_week = models.PositiveIntegerField(default=0)
+    # Controls whether a user will receive emails reminding them to stay on track with their learning goal
+    subscribed_to_goal_reminders = models.BooleanField(default=False)
```

Comment:
Similar naming comment here btw - that `goal` is kind of implied by the context it's in. But can leave it - no big deal.

Reply:
good point, removed to be consistent